### ISSUE TITLE
Click Outside of (dataset/model/project) after edit equivalent to click green checkmark

### ISSUE THREAD

**Issue by glenn-jocher** [ultralytics/hub-temp#9]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and found no similar feature requests.

### Description

User should be able to click anywhere outside of dataset/model/project that is being edited to automatically save changes and exit edit mode (i.e. the same effect as clicking the green checkmark). This would really increase the usability of the edit feature I think.

Basically what we did for dataset image preview applied here to save changes:

<img width="1579" alt="Screenshot 2021-11-20 at 12 48 14" src="https://user-images.githubusercontent.com/26833433/142725288-0198c69f-c300-4e3d-ac24-a12d8518c5d7.png">

**Comment by glenn-jocher** [comment]
I'm going to increase the scope here a little bit:

1. Same saving should apply to model configuration edits (i.e. click outside of configuration panel saves settings and edits configuration mode).
2. Hitting 'Enter' key on keyboard should have same effect as green checkmark also.

<img width="1551" alt="Screenshot 2021-11-20 at 12 58 55" src="https://user-images.githubusercontent.com/26833433/142725525-04d6202c-7b89-4429-a603-3a965879e24f.png">

**Comment by kalenmike** [comment]
Implemented for quick edits. Needs to be added to configuration.

**Comment by kalenmike** [comment]
Added to configuration. To be released on next deployment.

**Comment by glenn-jocher** [comment]
Confrim working well!
